src/g_canvas.c

```diff
@@ -914,14 +920,17 @@ void canvas_loadbangsubpatches(t_canvas *x)
     t_symbol *s = gensym("loadbang");
     for (y = x->gl_list; y; y = y->g_next)
         if (pd_class(&y->g_pd) == canvas_class)
-    {
-        if (!canvas_isabstraction((t_canvas *)y))
-            canvas_loadbangsubpatches((t_canvas *)y);
-    }
+        {
+            if (!canvas_isabstraction((t_canvas *)y))
```

Comment:
it seems that there are whitespace issues here.
do you mind replacing all the TABS with 4 SPACES?

Reply:
purpose because in Allman/BSD indentation style (which is used throughout Pd) brackets should be on the same level as the control structure - which in this case is the if-statement and not the for-loop. IMHO, putting the brackets on the same level as the for-loop is a bit weird and misleading. I can switch back to the original indentation if you think it's better style.

Reply:
no that's fine.
i was confused by the github diff display (I didn't check out the actual files; and now see that my comment was void).
please disregard my comment :-)